README.md

```diff
@@ -145,6 +145,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [SUBISU CABLENET (P.) LTD.](http://subisu.net.np/) | Jhamsikhel, Lalitpur | Internet, Network and Television Service Provider
 - [Sustainable Solutions](http://sussol.net/) | Pulchowk, Lalitpur | mSupply and MoneyWorks
 - [Tech Bizz Pvt. Ltd.](http://www.techbizznepal.com/) | Maitidevi, Kathmandu | Mobile Apps, Graphics Design, and Web Development Services
+- [TechNepal](https://technepal.io) | Online. | One stop tech news portal in Nepal. 
```

Comment:
Thank you for the contribution. Can you also add your location? @nkhanal0 

I get that you or your team might be working remotely but you might want to add in the location that you have registered or can relate to.

Reply:
I will do that. Thanks